module/documents/item.mjs

```diff
@@ -912,73 +953,88 @@ export default class Item5e extends Item {
    * @returns {object|boolean}             A set of data changes to apply when the item is used, or false.
    * @protected
    */
-  _getUsageUpdates({
-    consumeQuantity, consumeRecharge, consumeResource, consumeSpellSlot,
-    consumeSpellLevel, consumeUsage}) {
+  _getUsageUpdates(config) {
     const actorUpdates = {};
-    const itemUpdates = {};
+    let itemUpdates = {};
```

Comment:
This only needs to be `let` if the variable is re-assigned somewhere, but I can't see that happening in this method, unless I've missed it?